commons/src/main/java/io/hyscale/commons/utils/LogProcessor.java

```diff
@@ -37,7 +40,13 @@
 	private static final Logger logger = LoggerFactory.getLogger(LogProcessor.class);
 	private static final Integer DEFAULT_LINES = 100;
 
-	public void writeLogFile(InputStream is, String logFile) throws IOException {
+	public void writeLogFile(InputStream is, String logFile) throws IOException,HyscaleException{
+		if(is == null){
+			throw new HyscaleException(CommonErrorCode.INVALID_INPUTSTREAM,null);
+		}
+		if (StringUtils.isBlank(logFile)){
+			throw new HyscaleException(CommonErrorCode.FILE_NOT_FOUND,logFile);
```

Comment:
Actually you are checking for File path rather than file.
The exception message is not relevant to your check should be something like "file path found empty" or "empty file path found"